docs/README.md

```diff
@@ -156,6 +190,8 @@ If you want to run the oauth logout flow call:
 authenticator.logout()
 ```
 
+___When using Automated Auth no actual login endpoint is called. Instead local auth state is invalidated.___ 
```

Comment:
Isn't there some endpoint being called? It looks like `MicroHttpCLient` is used as part of Automated Auth to get a token from tokenUrl?

Reply:
my bad, this is for logout, but I wrote login endpoint in the note.. ill correct it to logout endpoint.
The micro client is used as you describe, url is in the config file